manifests/init.pp

```diff
@@ -99,19 +102,23 @@
   Variant[Boolean[false], Pattern[/icmp(v6|x)? type .+|tcp reset/]] $reject_with = 'icmpx type port-unreachable',
   Variant[Boolean[false], Enum['mask']] $firewalld_enable = 'mask',
   Optional[Array[Pattern[/^(ip|ip6|inet)-[-a-zA-Z0-9_]+$/],1]] $noflush_tables = undef,
+  Stdlib::AbsolutePath $nftables_configuration_path = '/etc/sysconfig/nftables.conf',
 ) {
   package { 'nftables':
     ensure => installed,
   } -> file_line {
     'enable_nftables':
       line   => 'include "/etc/nftables/puppet.nft"',
-      path   => '/etc/sysconfig/nftables.conf',
+      path   => $nftables_configuration_path,
       notify => Service['nftables'],
   } -> file {
     default:
       owner => 'root',
       group => 'root',
       mode  => '0640';
+    '/etc/nftables':
+      ensure => directory,
+      mode   => '0750';
```

Comment:
should we purge it? maybe only on debian?

Reply:
It seems reasonable to me that it be purged, although I do not know what gets put in the by other OSes.

Reply:
Upon some further reflexion, I'm not sure it is that useful to purge the directory since the nftables service is specifically using `/etc/nftables/puppet.nft`

Reply:
I guess, it's probably fine to not do. Just an idea that I wanted to throw in.